tutorials/btp-hana-xs-app-create/btp-hana-xs-app-create.md

```diff
@@ -0,0 +1,349 @@
+
+1.  In the **Project Explorer** view in the **SAP HANA Development** perspective in Eclipse, right-click your XS project, and choose **New** > **Other**.
+
+    **Tip**
+    > If your XS Project isn't visible in the **Project Explorer**, choose the **`︙`** button in menu bar for the **Project Explorer** view to change the project presentation from **Hierarchical** to **Flat**.
+
+2.  In the **Select a wizard** dialog, choose **SAP HANA** > **Application Development** > **XS JavaScript File**.
+
+3.  In the **New XS JavaScript File** dialog, enter **`MyFirstSourceFile.xsjs`** as file name.
+
+4.  Choose **Finish**.
+
+5.  In the `MyFirstSourceFile.xsjs` file, enter the following code and save the file:
+
+    `$.response.contentType = "text/html";`
+
+    `$.response.setBody( "Hello, World !");`
+
+    > By default, saving the file automatically commits the saved version of the file to the repository.
+
+    The example code shows how to use the SAP HANA XS JavaScript API's **response** object to write HTML. By typing **`$.`**, you have access to the API's objects.
+
+6.  Check that the application descriptor files (`.xsapp` and `.xsaccess`) are present in the root package of your new XS JavaScript application.
+
+    The application descriptors are mandatory and describe the framework in which an SAP HANA XS classic application runs.
+
+    The `.xsapp` file indicates the root point in the package hierarchy where content is to be served to client requests; the `.xsaccess` file defines who has access to the exposed content and how.
+
+
+    > By default, the project-creation Wizard creates the application descriptors automatically.  
+     If they're not present, you see a 404 error message in the Web Browser when you call the XS JavaScript service. In this case, you need to create the application descriptors manually. For more information, see the [SAP HANA Developer Guide for SAP HANA Studio](https://help.sap.com/viewer/product/SAP_HANA_PLATFORM/2.0.06/en-US?task=discover_task).
+
+7.  Open the context menu for the new file and select **Team** > **Activate**.
+
+  The activate operation publishes your work and creates the corresponding catalog objects. You can now test it.
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 5: ](Test your application)]
+
+Check if your application is working and if the `Hello, World!` message is displayed.
+
+1.  In the SAP BTP cockpit, choose **SAP HANA / SAP ASE** > **Databases & Schemas**.
+
+2.  Select the relevant SAP HANA tenant database in the list.
+
+3. Open the **SAP HANA Web-based Development Workbench** link under **Development Tools**.
+
+4. When the web editor opens, copy the first part of the URL up to and including the **"com"**.
+
+5. To form your target URL, add a **/** followed by the project name.
+
+6. Then add another **/** followed by the name of the file we just added in the previous step.
+
+    **Example**
```

Comment:
Change formatting to ""list"

Reply:
It is a numbered list already. What do you mean?